app/models/merchant.rb

```diff
@@ -25,4 +25,22 @@ def distinct_cities
     item_orders.distinct.joins(:order).pluck(:city)
   end
 
+  def change_status
+    if self.enabled?
+      self.update(enabled?: "false")
+    else
+      self.update(enabled?: "true")
+    end
+  end
+
+  def change_active_status
```

Comment:
I wonder if this might make more sense as an `item` class method and then in the controller, it would call it would call `merchant.items.change_status`?

Reply:
Ruthie - your comment makes a lot of sense. But then I looked at the merchant model...there are a number of methods in there that target either items or item_orders. Same with the item model and order model.

I don't understand convention well enough to know which methods belong in which model. Because we are already inconsistent about it, I would suggest merging the PR as is. We can refactor all the model methods later if we have time.